docs/topics/auth/default.txt

```diff
@@ -673,18 +673,78 @@ Django provides several views that you can use for handling login, logout, and
 password management. These make use of the :ref:`stock auth forms
 <built-in-auth-forms>` but you can pass in your own forms as well.
 
-Django provides no default template for the authentication views - however the
-template context is documented for each view below.
+Django provides no default template for the authentication views, you should
+create these yourself. The template context is documented in each view, see
```

Comment:
Django provides no default template for the authentication views, you should create these yourself. -> Django provides no default template for the authentication views. You should create your own templates for the views you want to use.